Source/Core/Token.cs

```diff
@@ -83,7 +88,48 @@ public int line
       get { return this._val; }
       set { this._val = value; }
     }
+    
+    public string /*!*/ leadingTrivia
+    {
+      get { return this._leadingTrivia; }
+      set { this._leadingTrivia = value; }
+    }
+    
+    public string /*!*/ trailingTrivia
+    {
+      get { return this._trailingTrivia; }
+      set { this._trailingTrivia = value; }
+    }
```

Comment:
Why do you have these backing fields instead of using an auto-implemented property?

Reply:
just to be consistent with the previous implementation.
If it was just me, I'd had public fields.